spec/ParseUser.spec.js

```diff
@@ -1789,7 +1789,7 @@ describe('Parse.User testing', () => {
     });
   });
 
-  it('should allow login with old authData token', done => {
+  it('should not allow login with old authData token', async () => {
```

Comment:
@mtrezza @dplewis should we change this? Any other comment for this PR?

Reply:
Maybe we can pick up the discussion in the advisory, I think there were some open questions what it means to change this?

Reply:
Should we add that as a breaking change?